ui/pages/confirm-add-suggested-nft/confirm-add-suggested-nft.js

```diff
@@ -0,0 +1,296 @@
+                          <Text
+                            variant={TextVariant.bodyMd}
+                            className="confirm-add-suggested-nft__nft-tokenId"
+                          >
+                            #{tokenId}
+                          </Text>
+                        </div>
+                      </div>
+                    </Box>
+                  );
+                }
+                return (
+                  <div
+                    className="confirm-add-suggested-nft__nft-list-item"
+                    key={`${address}-${tokenId}`}
+                  >
+                    <div
+                      className={classnames(
+                        'confirm-add-suggested-nft__nft-details',
+                      )}
+                    >
+                      {nftImageURL ? (
+                        <img
+                          className="confirm-add-suggested-nft__nft-image"
+                          src={nftImageURL}
+                          alt={name || tokenId}
+                        />
+                      ) : (
+                        <NftDefaultImage className="confirm-add-suggested-nft__nft-image-default" />
+                      )}
+                      <div className="confirm-add-suggested-nft__nft-sub-details">
+                        {rpcPrefs.blockExplorerUrl ? (
+                          <ButtonLink
+                            className="confirm-add-suggested-nft__nft-name"
+                            href={blockExplorerLink}
+                            title={address}
+                            target="_blank"
+                            size={BUTTON_SIZES.INHERIT}
+                          >
+                            {name || symbol || shortenAddress(address)}
+                          </ButtonLink>
+                        ) : (
+                          <Text
+                            variant={TextVariant.bodySm}
+                            className="confirm-add-suggested-nft__nft-name"
+                            title={address}
+                          >
+                            {name || symbol || shortenAddress(address)}
+                          </Text>
+                        )}
+                        <Text
+                          variant={TextVariant.bodySm}
+                          className="confirm-add-suggested-nft__nft-tokenId"
+                        >
+                          #{tokenId}
+                        </Text>
+                      </div>
+                    </div>
+                    <ButtonIcon
+                      className="confirm-add-suggested-nft__nft-remove"
+                      data-testid={`confirm-add-suggested-nft__nft-remove-${id}`}
+                      iconName={IconName.Close}
+                      size={ButtonIconSize.Sm}
+                      color={IconColor.iconMuted}
+                      onClick={(e) => {
+                        console.log('rejecting');
```

Comment:
console.log should be removed

Reply:
done [here](https://github.com/MetaMask/metamask-extension/pull/19454/commits/19234ca62e07ac3b5a83f7181f4f946e68100b43)